config.sh

```diff
@@ -63,15 +63,15 @@ function build_osx_wheel {
 function run_tests {
     # Runs tests on installed distribution from an empty directory
     # OSX tests seem to time out pretty often
-    if [ -z "$IS_OSX" ]; then
+    if [[ -z "$IS_OSX" && `uname -m` != 'aarch64' ]]; then
         local testmode="full"
     else
         local testmode="fast"
     fi
     # Check bundled license file
     python ../check_installed_package.py
     # Run tests
-    python ../run_scipy_tests.py $testmode -- -n2 -rfEX
+    python ../run_scipy_tests.py $testmode -- -n8 -rfEX
```

Comment:
I tried restarting the single travis CI matrix entry failure which is this:

`worker 'gw5' crashed while running 'signal/tests/test_signaltools.py::TestConvolve2d::test_large_array'`

This makes me wonder if this `-n8` multi-core test setting is being applied to all jobs and not just the ARM64 job? If so, we'd probably want to revert that--I think regular Travis for x86_64 tends to provide ~2 cores? Maybe that's part of the reason for that strange crash?

Reply:
To clarify, can we just bump up the number of cores used for ARM only?

Reply:
Now bumping up the number of cores used for ARM only.